src/styles/StyleUtils.ts

```diff
@@ -259,9 +259,12 @@ function getSafeAreaMargins(insets?: EdgeInsets): ViewStyle | CSSProperties {
 
 function getZoomCursorStyle(isZoomed: boolean, isDragging: boolean): ViewStyle | CSSProperties {
     if (!isZoomed) {
+        // TODO: Remove this "eslint-disable-next" once the theme switching migration is done and styles are fully typed
```

Comment:
Let's this issue link to these comments:
https://github.com/Expensify/App/issues/27337

Reply:
Let's link this issue to these comments* 
wow I think I might be tired 😂